Loop/Managers/DeviceDataManager.swift

```diff
@@ -1530,6 +1531,14 @@ struct CancelTempBasalFailedError: LocalizedError {
     }
 }
 
+//MARK: - RemoteDataServicesManagerDelegate protocol conformance
+
+extension DeviceDataManager : RemoteDataServicesManagerDelegate {
+    var shouldSyncToRemoteService: Bool {
+        return cgmManager?.shouldSyncToRemoteService ?? false
```

Comment:
If we don't have a cgmmanager, but do have glucose values for some reason (unlikely, but possible), then we should probably still upload them.

Reply:
Good point - done.